blocks/embed/embed.css

```diff
@@ -1,20 +1,14 @@
-.embed {
+.embed.matterport {
```

Comment:
I'm struggling with this one _(my own mental battle)_, if the embed should know about the matterport at all.  If the matterport is used on the inventory page, then only the inventory layout should specify the styles to specify the styles specific to the matterport being used on that page?  My struggle isn't around the applied styles, its about where this style should be defined.